### PR TITLE
Added unit testcases for sparkline chart.

### PR DESCRIPTION
**Pull request by v-asurse**
Added unit testcases for sparkline chart.
Note: Not able to run ''yarn change" because node version incompatibility.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-charting/UnitTests/SparklineChartUT.test.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { getByClass, testWithWait } from '../src/utilities/TestUtility.test';
```
Comment (fabricteam) [inline packages/react-charting/UnitTests/SparklineChartUT.test.tsx:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31843&commitId=2b7c3ab57881bf3c63845a897439b7551fa01171&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/docs/TestPlans/SparklineChart/UnitTests.md
Comment (AtishayMsft) [inline packages/react-charting/docs/TestPlans/SparklineChart/UnitTests.md:1]: The file is not showing properly in preview mode.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**